Installation Graphics Update (Fixed)

KillerDroid96 opened the pull request:
-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed
-Added Kernel Version (Uncomment and change path to use)
-Removed Manufacture and replaced with Codename (ex. dumpling)
-Changed ROM Version to crDroid Version (May revert this)

KillerDroid96 committed:
Installation Graphics Update (Fixed)

-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed
-Added Kernel Version (Uncomment and change path to use)
-Removed Manufacture and replaced with Codename (ex. dumpling)
-Changed ROM Version to crDroid Version (May revert this)

gwolf2u reviewed (changes requested):
remove unneeded code comments 
ex: line 860

KillerDroid96 committed:
Installation Graphics Update (Cleaned Up)

-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed
-Removed Manufacture and replaced with Codename (ex. dumpling)
-Changed ROM Version to crDroid Version (May revert this)
-Removed Kernel Comments

To Add Kernel:
Change USER & BUILD
kernel = ("/home/$USER/$BUILD/out/target/product/dumpling/obj/KERNEL_OBJ/include/config/kernel.release")
kernelver = open(kernel).read()
script.Print(" Kernel Version   : %s"%(kernelver));

KillerDroid96 committed:
Display crDroid Banner at end of build
